myClass.ts

```diff
@@ -0,0 +1,48 @@
+class Myclass {
+	firstNumber: number = 0;
+	lastNumber: number = 0;
+	backend: string = "http://localhost:2050";
+	resizeTimeout: number = 0;
```

Comment:
This value is not being used in this class... Maybe you forgot to add methods that uses it? or maybe it should not be in this class?